### PR TITLE
EKS cluster deployment

### LINKED ISSUE #274: Kubernetes deployment guide
Deploying a Kubernetes cluster is a complicated endeavor given the miriad of managed Kubernetes' services.

If our images and/or manifests does have any requirements a kubernetes deployment should comply with, we need to specify in our documents and guides.

This will help our user determine the best configuration for their kubernetes' services to be compatible with what our code here needs to work.

We need to clarify the impact the use of certificates has in the communications between  wazuh-indexer and wazuh-dashboard when using the service address to communicate them,  the security groups and firewall rules suggested, etc.

### PR DESCRIPTION
**Pull request by teddytpc1**
This PR closes #274.

### CODE REVIEW

**Review by vcerenu — DISMISSED** [review]
LGTM

File: deploy_eks_cluster/README.md [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+- Install `AWS CLI`: https://docs.aws.amazon.com/cli/latest/userguide/getting-started-install.html
+- Configure `AWS CLI`: https://docs.aws.amazon.com/cli/latest/userguide/getting-started-quickstart.html
+- Install `eksctl`: https://docs.aws.amazon.com/eks/latest/userguide/eksctl.html
+- Install `kubectl`: https://kubernetes.io/docs/tasks/tools/#kubectl
+
+## Deployment
+
+To deploy an EKS cluster, you need to:
+
+1. Define whether you will use an existing VPN.
```
Comment (okynos) [inline deploy_eks_cluster/README.md:36]:
```suggestion
1. Define whether you will use an existing VPC.
```

**Review by okynos — CHANGES_REQUESTED** [review]
Really cool stuff, minor change, thanks.

**Review by okynos — DISMISSED** [review]

File: deploy_eks_cluster/README.md [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+## YAML files
+
+* `cluster.yml` should be used if the creation of the VPC and subnets is required.
+* `cluster_existing_vpc.yml` should be used if the VPC and subnets already exist.
+
+
+## AWS Resources
+
+The goal of this section is to simplify the creation of an EKS cluster.
+The execution of this guide will create 2 Cloudformation stacks with the following resources:
```
Comment (gdiazlo) [inline deploy_eks_cluster/README.md:14]: Why are we using two cloudformation stacks, all the resources are duplicated on each stack. We need to explain the motivation behind this setup.

Comment (gdiazlo) [reply deploy_eks_cluster/README.md:14]: Also, we need to describe what instances we are using and why.

**Review by gdiazlo — CHANGES_REQUESTED** [review]
I think we need to include more information in the README. The goal of the issue was to let our users know how and why we do this to test Wazuh in a EKS cluster.

File: deploy_eks_cluster/README.md [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+
+## YAML files
+
+* `cluster.yml` should be used if the creation of the VPC and subnets is required.
+* `cluster_existing_vpc.yml` should be used if the VPC and subnets already exist.
+
+
+## AWS Resources
+
+The goal of this section is to simplify the creation of an EKS cluster.
```
[the remark this answers is not in the document]
Comment (teddytpc1) [reply deploy_eks_cluster/README.md:13]: You are right. I will move it.

**Review by vcerenu — APPROVED** [review]
LGTM

**Review by gdiazlo — APPROVED** [review]
lgtm

**Comment by cyberresearch-us** [comment]
Just curious if/when this will be merged.

**Review by c-bordon — APPROVED** [review]
LGTM

**Pull request merged**